src/khepri_machine.erl

```diff
@@ -1303,6 +1328,22 @@ apply(
     State1 = set_projections(State, ProjectionTree1),
     Ret = {State1, Reply},
     post_apply(Ret, Meta);
+apply(Meta, #unregister_all_projections{}, State) ->
+    ProjectionTree = get_projections(State),
+    Projections = khepri_pattern_tree:fold(
+                   ProjectionTree,
+                   fun(PathPattern, Projections, Acc) ->
+                           lists:foldl(
+                             fun(Projection, Acc1) ->
+                                     khepri_projection:delete(Projection),
```

Comment:
I’m wondering, should we protect the code against an exception from ETS and return an error? It could take down the Ra server otherwise.

Reply:
We should add a `catch` in `khepri_projection:delete/1` but the existing callers (and this call here) should ignore the error I think. If we get an error that means that the table doesn't exist which is the state we want it in. I don't think that should be possible (at this point the table should exist) but it would be nice to protect against something unexpected killing the server here